worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -11,163 +11,19 @@ namespace RTC
 {
 	/* Static. */
 
+	// Limit retransmission buffer max size to 2500 items.
+	static constexpr size_t RetransmissionBufferMaxItems{ 2500u };
```

Comment:
This is the retransmission buffer max size (passed to `RetransmissionBuffer` class as argument in the constructor. In my tests this is good enough.